common/supplemental/supplementalData.xml

```diff
@@ -4535,30 +4535,35 @@ XXX Code for transations where no currency is involved
         <calendar type="islamic">
           <calendarSystem type="lunar" />
           <eras>
+              <era type="-1" end="622-7-14" code="islamic-inverse" aliases="bh"/>
```

Comment:
I gave it the code "islamic-inverse" because I want to land the code updates, #4519, atomically once we have agreement on them.